lib/LaTeXML/Package/latexml.sty.ltxml

```diff
@@ -34,8 +34,10 @@ DeclareOption('noids', sub { AssignValue('GENERATE_IDS' => 0, 'global'); });
 DeclareOption('comments',   sub { AssignValue('INCLUDE_COMMENTS' => 1, 'global'); });
 DeclareOption('nocomments', sub { AssignValue('INCLUDE_COMMENTS' => 0, 'global'); });
 
-DeclareOption('profiling',   sub { AssignValue('PROFILING' => 1, 'global'); });
-DeclareOption('noprofiling', sub { AssignValue('PROFILING' => 0, 'global'); });
+DeclareOption('tracing',     sub { AssignValue(TRACING => (LookupValue('TRACING') || 0) | 3); });
```

Comment:
Could we document the bitmask details here (in latexml.sty)? Especially since it is likely to be ported to an `enum` type, it would be nice to match the implied intention.